readthedocs/templates/projects/notifications/deprecated_config_file_used_email.txt

```diff
@@ -0,0 +1,9 @@
+{% extends "core/email/common.txt" %}
+{% block content %}
+Your project "{{ project.slug }}" is not using a <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">.readthedocs.yaml</a> configuration file and will stop working soon.
+
+We strongly recommend you to add a configuration file to keep your builds working.
```

Comment:
I'd match the text above here. Namely, we're not recommending this, we are telling the user that their build _will absolutely stop building_.

```suggestion
Configuration files will <strong>soon be required</strong> by projects, and will no longer be optional.
You will need to add a configuration file to your project to ensure your project continues building successfully.
```